packages/account/src/Components/terms-of-use/terms-of-use.scss

```diff
@@ -54,3 +58,27 @@
         }
     }
 }
+
+.fatca-declaration {
+    &__layout {
+        display: flex;
+        gap: 1.6rem;
+        flex-direction: column;
+    }
+
+    &__points {
+        line-height: 2rem !important;
+
+        @include mobile {
+            line-height: 1.8rem !important;
+        }
+    }
+
+    &__agreement {
+        width: 32.8rem !important;
+
+        .dc-dropdown__container {
+            margin-bottom: 2rem !important;
+        }
+    }
+}
```

Comment:
can we please get ridof these important rules?

Reply:
Some are required to over ride default styles of Text component and field width